client/js/_author/components/assessments/question_types/audio_upload.jsx

```diff
@@ -20,6 +21,24 @@ export default class AudioUpload extends React.Component {
     return seconds;
   }
 
+  static rangeWarning() {
+    return (
+      <span className="author--c-inline-error">Please enter a positive number under 240</span>
+    );
+  }
+
+  constructor(props) {
+    super(props);
+    const timeLimit = AudioUpload.getAudioLimit(props.item)
+      || `${AudioUpload.MAX_TIME}`;
+    const displayWarning = timeLimit > AudioUpload.MAX_TIME;
```

Comment:
Or more applicably `"2000" > "240"` returns false (this is very interesting, I had to play with this concept)